cmd/odf/operator/rook/set.go

```diff
@@ -0,0 +1,17 @@
+package rook
+
+import (
+	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/spf13/cobra"
+)
+
+var setCmd = &cobra.Command{
+	Use:     "set",
+	Short:   "Set the property in the rook-ceph-operator-config configmap.",
+	Example: "odf operator rook set <KEY> <VALUE>",
```

Comment:
```suggestion
	Example: "odf operator rook set <KEY> <VALUE>",
```

@travisn does this command structure sounds good?

Reply:
Yes, lgtm

Reply:
great